src/art/AashiGoel/Halloween.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nosifer">
+    </head>
+<body>
+<h2>Happy Halloween!!!</h2>
+<img src="Halloween.jpg" alt="pumpkin">
```

Comment:
Please provide the license to the image. Images are usually under copyright, so we need to make sure that we can showcase this without violating copyright.

Reply:
I downloaded this image from google and I can't find the exact image on searching now.....

Reply:
https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.cre8iveskill.com%2Fshop%2Fqueen-mummy-vector-art&psig=AOvVaw3HcBVWYOBiowS2F9r9b-8K&ust=1666113269633000&source=images&cd=vfe&ved=0CA0QjRxqFwoTCMjgnN_h5_oCFQAAAAAdAAAAABAE
This is the image I downloaded

Reply:
Downloading images from Google doesn't give you copyright.

I suggest that you replace the image with something from unsplash, which has images with permissive licenses.

Reply:
okay I will :)